Client/src/Components/TabPanels/Status/GeneralSettingsPanel.jsx

```diff
@@ -0,0 +1,281 @@
+							</Typography>
+						</Stack>
+					</Box>
+					<Stack gap={theme.spacing(6)}>
+						<Checkbox
+							id="published-to-public"
+							label={`Published and visible to the public`}
+							isChecked={localData.publish}
+							value={"localData.publish"}
+							onChange={handleChange}
+							onBlur={handleBlur}
+						/>
+					</Stack>
+				</ConfigBox>
+
+				<ConfigBox>
+					<Box>
+						<Stack gap={theme.spacing(6)}>
+							<Typography component="h2">Basic Information</Typography>
+							<Typography component="p">
+								Define company name and the subdomain that your status page points to.
+							</Typography>
+						</Stack>
+					</Box>
+					<Stack gap={theme.spacing(18)}>
+						<TextInput
+							id="companyName"
+							type="text"
+							label="Company name"
+							value={localData.companyName}
+							onChange={handleChange}
+							onBlur={handleBlur}
+							helperText={errors["companyName"]}
+							error={errors["companyName"] ? true : false}
+						/>
+						<TextInput
+							id="url"
+							type="url"
+							label="SubURL"
+							value={localData.url}
+							startAdornment={<HttpAdornment https={true} />}
+							//prefix={"http://uptimegenie.com/"}
+							onChange={handleChange}
+							onBlur={handleBlur}
+							helperText={errors["url"]}
+							error={errors["url"] ? true : false}
+						/>
+					</Stack>
+				</ConfigBox>
+				<ConfigBox>
+					<Box>
+						<Stack gap={theme.spacing(6)}>
+							<Typography component="h2">Appearance</Typography>
+							<Typography component="p">
+								Define the default look and feel of your public status page.
+							</Typography>
+						</Stack>
+					</Box>
+					<Stack gap={theme.spacing(6)}>
+						<Select
+							id="display-timezone"
+							label="Display timezone"
+							value={localData.timezone}
+							onChange={handleChange}
+							onBlur={handleBlur}
+							items={timezones}
+							error={errors["display-timezone"]}
+						/>
+						<ImageField
+							id="logo"
+							src={localData.logo?.src ?? logo?.src}
+							loading={progress.isLoading && progress.value !== 100}
+							onChange={handleLogo}
+						/>
+						{progress.isLoading || progress.value !== 0 || errors["logo"] ? (
+							<ProgressUpload
+								icon={<ImageIcon />}
+								label={logo?.name}
+								size={formatBytes(logo?.size)}
+								progress={progress.value}
+								onClick={removeLogo}
+								error={errors["logo"]}
+							/>
+						) : (
+							""
+						)}
+						<TextInput
+							id="color"
+							label="Color"
+							value={localData.color}
+							error={errors["color"]}
+							onChange={handleChange}
+							onBlur={handleBlur}
+						/>
+						<Select
+							id="theme"
+							label="Theme"
+							value={localData.theme}
+							onChange={handleChange}
+							onBlur={handleBlur}
+							items={[
+								{ _id: "light", name: "Light" },
+								{ _id: "dark", name: "Dark" },
+							]}
```

Comment:
I believe we discussed removing this as dark/light theme is set elsewhere in the app

Reply:
Fixed in : Fixed here: https://github.com/bluewave-labs/checkmate/pull/1307/commits/c301656a7aeb61a6739d4510f1587932dd81bfd4